cpp/src/traversal/bfs_kernels.cuh

```diff
@@ -92,7 +92,7 @@ __global__ void fill_unvisited_queue_kernel(int *visited_bmap,
     // saving the common offset
     if (threadIdx.x == (FILL_UNVISITED_QUEUE_DIMX - 1)) {
       IndexType total               = unvisited_thread_offset + n_unvisited_in_int;
-      unvisited_common_block_offset = atomicAdd(unvisited_cnt, total);
+      unvisited_common_block_offset = traversal::atomicAdd(unvisited_cnt, total);
```

Comment:
Ideally, traversal::atomicAdd is not specific to traversal and better go to RAFT (e.g. https://github.com/rapidsai/raft/blob/branch-0.15/cpp/include/raft/cudart_utils.h)

If it is too much for this PR, better add a comment like
```
// FIXME: this should better go to RAFT
```

Reply:
Deferring the move to RAFT for now since RAFT is a separate repo. Added the comment.